src/utils/src/main/python/java_params_patch.py

```diff
@@ -0,0 +1,38 @@
+from pyspark.ml.wrapper import JavaParams
+
+@staticmethod
+def _mml_from_java(java_stage):
+    """
+    Given a Java object, create and return a Python wrapper of it.
+    Used for ML persistence.
+
+    Meta-algorithms such as Pipeline should override this method as a classmethod.
+    """
+    def __get_class(clazz):
+        """
+        Loads Python class from its name.
+        """
+        parts = clazz.split('.')
+        module = ".".join(parts[:-1])
+        m = __import__(module)
```

Comment:
this is verbatim from spark, just with an additional line. I tested it in python3